### PR TITLE
Add development instructions to README

### LINKED ISSUE #16: Add development info to README
Currently the steps include:
* Clone repo
* Update submodules
* Install requirements
* Export variables

Then on each step:
* Install stubs
* Run pytest

### PR DESCRIPTION
**Pull request by lewisjb**
Fixes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,21 @@ stable version as:
...
+git submodule update --init --recursive
+pip install -r external/mypy/test-requirements.txt
+pip install -e external/mypy
+pip install -e .
+```
+
+Then, to run the tests, simply:
+```
+pytest
+```
```
Comment (ilevkivskyi) [inline README.md]: Does it actually work like this? I think one needs to  `export PYTHONPATH=external/mypy` before running the tests. Or this is not needed if mypy is installed?

Comment (lewisjb) [reply README.md]: Yeah, installing mypy avoids needing to change the `PYTHONPATH`, which has the added benefit of the instructions working across most platforms

**Comment by lewisjb** [comment]
I just signed the CLA, thanks for pointing it out; I had missed it

**Comment by ilevkivskyi** [comment]
Great, thanks!

**Pull request merged**